### ISSUE TITLE
Sleect * INTO x from Y

### ISSUE THREAD

**Issue by jimsandersonuk** [sqlkata/querybuilder#218]
Can we have a statement that allows us to insert into a temp table using the sql standard INTO?

All other statements seem to be available but this, and hand-coding this goes against the use-case for Kata.

cheers,

Jim

**Comment by ahmad-moussawi** [comment]
No, currently you can use the `db.Statement()` method, but it's nice to have, 
can you provide a full example, on how it should be implemented on multiple dbs?
so you can speed up the process a bit

**Comment by mawroos** [comment]
Hi, 
In SQL Server, you can do this

` SELECT * 
       into equipments_backup
	FROM test
    where id = 2
`

Postgres SQL
`SELECT * INTO films_recent FROM films WHERE date_prod >= '2002-01-01';`


MySQL,SQLLite
'CREATE TABLE equipments_backup 
AS SELECT * 
	FROM test
    where id = 2
 ' 

in sqlKata, can do something like 
`db.Query("Books").WhereTrue("IsPublished").Into("NewTableName")`

**Comment by rolivares** [comment]
yes it's a very useful statement, I came here searching about "how to in sqlkata" ... I'll try a different solution for my process
